src/main/java/com/dataguise/saas/dto/SourcesDetails.java

```diff
@@ -0,0 +1,97 @@
+package com.dataguise.saas.dto;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+public class SourcesDetails {
+    private int sourceId;
+    private String sourceName;
+    private String sourceType;
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocVariableCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/dto/SourcesDetails.java L9
    
    Missing a Javadoc comment.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1842711632>*